marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.test.tsx

```diff
@@ -55,10 +55,18 @@ describe("Mapping Card component", () => {
       getByRole = renderResults.getByRole;
       queryAllByRole = renderResults.queryAllByRole;
       getByText = renderResults.getByText;
+      getByTestId = renderResults.getByTestId;
     });
 
     expect(getByRole("edit-mapping")).toBeInTheDocument();
+    fireEvent.mouseOver(getByRole('edit-mapping'));
+    await wait (() => expect(getByText('Edit')).toBeInTheDocument());
+    expect(getByTestId("Mapping1-stepDetails")).toBeInTheDocument();
+    fireEvent.mouseOver(getByTestId('Mapping1-stepDetails'));
+    await wait (() => expect(getByText('Step Details')).toBeInTheDocument());
     expect(getByRole("settings-mapping")).toBeInTheDocument();
```

Comment:
Same thing here. Let's remove all such redundant steps where we are validating an element first in the document and then trying to mouseover it in separate step since the mouseover step already does the trick.